### PR TITLE
Support for Camembert Huggingface bert-like models

### PR DESCRIPTION
**Pull request by itzsimpl**
Signed-off-by: Iztok Lebar Bajec <ilb@fri.uni-lj.si>

# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: nlp

# Changelog 
- Support for loading Huggingface Camembert models, e.g. https://huggingface.co/camembert-base

# Usage
* You can potentially add a usage example below
In `/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml` set
```python
pretrained_model_name: camembert-base
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/nlp/modules/common/huggingface/camembert.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 The Google AI Language Team Authors and
+# The HuggingFace Inc. team.
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```
Comment (ekmb) [inline nemo/collections/nlp/modules/common/huggingface/camembert.py:3]: Could you change this line to `# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.`

File: nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py [hunk 2]
```diff
@@ -78,6 +87,7 @@
     'RobertaTokenizer': "vocab.json",
     'BertTokenizer': "vocab.txt",
     'DistilBertTokenizer': "vocab.txt",
+    'CamembertTokenizer': "sentencepiece.bpe.model",
```
Comment (ekmb) [inline nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py:90]: How about using the same name as in AlbertTokenizer, as they both use SentencePiece model, i.e. `"spiece.model"`?

**Review by ekmb — CHANGES_REQUESTED** [review]
Thanks for your contribution! 
Please run `cd NeMo/ && python setup.py style --fix` to fix style errors.

File: nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py (hunk 2 above)
Comment (itzsimpl) [reply nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py:90]: I used the naming convention as present in Camembert model files, see for example https://huggingface.co/camembert-base/tree/main, https://huggingface.co/EMBEDDIA/sloberta/tree/main, https://huggingface.co/cmarkea/distilcamembert-base/tree/main. This seems to differ from Albert models, which use `spiece.model`, see https://huggingface.co/albert-base-v2/tree/main. 

I suggest leaving as is.

**Comment by itzsimpl** [comment]
As for running `cd NeMo/ && python setup.py style --fix` the command requires `isort` (not in NeMo `requirements.txt` and not customarily installed, so a heads up would be welcome). After installing it, it still fails with the following:
```bash
$ python setup.py style --fix
running style
Running command: isort --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=119 -rc -ws .
ERROR: Found nemo_text_processing.text_normalization.normalize import while parsing, but FIRSTPARTY was not included in the `sections` setting of your config. Please add it before continuing
See https://pycqa.github.io/isort/#custom-sections-and-ordering for more info.
Running command: black --skip-string-normalization --line-length=119 .
error: [Errno 2] No such file or directory: 'black'
```
Since none of these were modified by this PR I left everything as is.

**Comment by ekmb** [comment]
> 
Right, the CI is failing due to the style issues of the newly added files. Have you installed NeMo by running [`./reinstall.sh`](https://github.com/NVIDIA/NeMo/tree/stable#from-source)?
black/isort is a part of https://github.com/NVIDIA/NeMo/blob/main/requirements/requirements_test.txt

**Comment by itzsimpl** [comment]
Ok, thnx, I was using `pip install -e .`, had to run `./reinstall.sh` inside a container and then in it `python setup.py style --fix`.

**Review by ekmb — APPROVED** [review]

**Comment by ekmb** [comment]
@itzsimpl, thanks for your PR!

**Pull request merged**